wiki/Bancho/th.md

```diff
@@ -0,0 +1,5 @@
+# Bancho (เซิฟเวอร์แชท)
+
```

Comment:
the original article has tags and other metadata, please copy it over and, if applicable, translate the tags

Reply:
Added the tags